docs/tutorial/maps.rst

```diff
@@ -181,748 +166,386 @@ Some basic statistical functions are built into Map objects:
+        (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+        (-1228.76466158, -1224.62447509)>
+    >>> my_map.top_right_coord  # doctest: +REMOTE_DATA
+    <SkyCoord (Helioprojective: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+        (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+        (1235.21095899, 1227.39598836)>
 
-This creates a plot window with all axes defined, a plot title, and the image of the map data defined by the contents of the map.
-In non-interactive mode the plot window blocks execution and must be closed before doing anything else.
+But what if we wanted to know what pixel these physical coordinates correspond to?
+Thankfully, each Map has an associated World Coordinate System, or WCS, which is derived from the underlying metadata and expressed as an `astropy.wcs.WCS` object.
+The WCS is accessible as an attribute:
 
-plot()
-------
+.. code-block:: python
 
-For more advanced plotting the base sunpy objects also provide a `~sunpy.map.mapbase.GenericMap.plot` command.
-This command is similar to the pyplot `~matplotlib.pyplot.imshow` command in that it will create a figure and axes object for you if you haven't already.
+    >>> my_map.wcs  # doctest: +REMOTE_DATA
+    WCS Keywords
+    <BLANKLINE>
+    Number of WCS axes: 2
+    CTYPE : 'HPLN-TAN'  'HPLT-TAN'
+    CRVAL : 0.00089530541880571  0.00038493926472939
+    CRPIX : 512.5  512.5
+    PC1_1 PC1_2  : 0.99999706448085  0.0024230207763071
+    PC2_1 PC2_2  : -0.0024230207763071  0.99999706448085
+    CDELT : 0.00066744222222222  0.00066744222222222
+    NAXIS : 1024  1024
 
-When you create a plot with `~sunpy.map.GenericMap.peek` or
-`~sunpy.map.GenericMap.plot`, sunpy will use `astropy.visualization.wcsaxes` to
-represent coordinates on the image accurately, for more information see
-:ref:`wcsaxes-plotting`.
+WCS is a fairly complex topic, but all we need to know for now is that the WCS provides the transformation between the pixel coordinates of the image and physical or "world" coordinates.
+In particular, we will only focus on two methods: `~astropy.wcs.WCS.world_to_pixel` and `~astropy.wcs.WCS.pixel_to_world`.
+First, let's find the pixel location corresponding to the center of the Map,
 
-Using `~sunpy.map.GenericMap.plot` it is possible to customise the look of the
-plot by combining sunpy and matplotlib commands, for example you can over plot
-contours on the Map:
+.. code-block:: python
 
-.. plot::
-    :include-source:
+    >>> center_pixel = my_map.wcs.world_to_pixel(my_map.center)  # doctest: +REMOTE_DATA
+    >>> center_pixel  # doctest: +REMOTE_DATA
+    (array(511.5), array(511.5))
 
-    import matplotlib.pyplot as plt
-    import astropy.units as u
+Notice that these coordinates are not necessarily integers.
```

Comment:
Do we need to explain here, or before, that integers correspond to the lower left corners of pixels, and non-integers are locations within a pixel?

Reply:
I wondered about that...In the last code snippet in this example, I do state that (0,0) is the center of the lower-left corner pixel. I think this information is important, but I'm worried about it being out of place in a Map tutorial. My preference would be to write a topic guide on pixel coordinate conventions (including diagrams!) and then to refer to that here.

Reply:
hmm I would even suggest defining a new SkyCoord and doing that here instead of using the map center? or show both

Reply:
Why define a new one? The center/corner attributes are demonstrated above and the user has (presumably) already read the coordinates section of the tutorial which demonstrates how to create a SkyCoord.